### PR TITLE
Export exceptions to the resource file with localization in ukrainian and english

### PR DESCRIPTION
**Pull request by ValDekh**
**As a** user **I want to** receive localized (English or Ukrainian) and standardized messages on the UI **so that** I can understand it properly.

**Description**
As the project with it's complexity grows we need to add localization and refactor existing messages to be read from one source and to be standardized.

The following steps must be completed as part of this ticket:

- Create new class library project with resource files - Streetcode.Resources;
- Add resource files with English and Ukrainian localizations;
- Move all existing application messages to resource files;
- Implement reading messages from resource files based on user's localization;

### Acceptance Criteria

- User should receive messages based on language set on his browser (English or Ukrainian);
- Messages are standardized (e.g. for different not found messages should be the same message structure, so shouldn't be: 1) "The streetcode with id - 4 is not found" and 2) "We could not find the news with id - 4" - the message should have the same structure);

**Epic link**
E.g.: Epic #100 [Epic](https://jira.softserve.academy/browse/100)

**Labels to be added**
"User story", Correspondind Functionality (e.g. User), Priority ("pri: "), "Corresponding Epic"

### Tasks 
1. - [x] Development
2. - [x] Unit tests
3. - [ ] Documentation
4. - [ ] QA

### CODE REVIEW

File: Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Tag/GetTagByTitle/GetTagByTitleHandler.cs [hunk 1]
```diff
@@ -12,13 +14,13 @@ public class GetTagByTitleHandler : IRequestHandler<GetTagByTitleQuery, Result<T
 {
     private readonly IMapper _mapper;
     private readonly IRepositoryWrapper _repositoryWrapper;
-    private readonly ILoggerService _logger;
+    private readonly IStringLocalizer<CannotFindSharedResource> _stringLocalizerCannotFind;
 
-    public GetTagByTitleHandler(IRepositoryWrapper repositoryWrapper, IMapper mapper, ILoggerService logger)
```
Comment (Tysyatsky) [inline Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Tag/GetTagByTitle/GetTagByTitleHandler.cs:17]: Here you can see the logger replacement.

File: Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Tag/GetTagByTitle/GetTagByTitleHandler.cs [hunk 2]
```diff
@@ -27,8 +29,7 @@ public async Task<Result<TagDTO>> Handle(GetTagByTitleQuery request, Cancellatio
 
         if (tag is null)
         {
-            string errorMsg = $"Cannot find any tag by the title: {request.Title}";
-            _logger.LogError(request, errorMsg);
```
Comment (Tysyatsky) [inline Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Tag/GetTagByTitle/GetTagByTitleHandler.cs:31]: and here

File: Streetcode/Streetcode.BLL/MediatR/Analytics/StatisticRecord/Delete/DeleteStatisticRecordHandler.cs [hunk 3]
```diff
@@ -11,13 +10,11 @@ public class DeleteStatisticRecordHandler : IRequestHandler<DeleteStatisticRecor
     {
         private readonly IMapper _mapper;
         private readonly IRepositoryWrapper _repositoryWrapper;
-        private readonly ILoggerService _logger;
 
-        public DeleteStatisticRecordHandler(IMapper mapper, IRepositoryWrapper repositoryWrapper, ILoggerService logger)
+        public DeleteStatisticRecordHandler(IMapper mapper, IRepositoryWrapper repositoryWrapper)
```
Comment (Tysyatsky) [inline Streetcode/Streetcode.BLL/MediatR/Analytics/StatisticRecord/Delete/DeleteStatisticRecordHandler.cs:14]: and here

**Review by Tysyatsky — CHANGES_REQUESTED** [review]
I am so sorry to tell you that your changes with string localizer overlapped the logger implementation. So, you need to check where the violation occurs and fix it properly. There are some screenshots.

image

image

image

File: Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Tag/GetTagByTitle/GetTagByTitleHandler.cs (hunk 1 above)
Comment (ValDekh) [reply Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Tag/GetTagByTitle/GetTagByTitleHandler.cs:17]: done

File: Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Tag/GetTagByTitle/GetTagByTitleHandler.cs (hunk 2 above)
Comment (ValDekh) [reply Streetcode/Streetcode.BLL/MediatR/AdditionalContent/Tag/GetTagByTitle/GetTagByTitleHandler.cs:31]: done

File: Streetcode/Streetcode.BLL/MediatR/Analytics/StatisticRecord/Delete/DeleteStatisticRecordHandler.cs (hunk 3 above)
Comment (ValDekh) [reply Streetcode/Streetcode.BLL/MediatR/Analytics/StatisticRecord/Delete/DeleteStatisticRecordHandler.cs]: done

File: Streetcode/DbUpdate/Properties/launchSettings.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  "profiles": {
+    "DbUpdate": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "https://localhost:59584;http://localhost:59586"
+    }
+  }
```
Comment (Tysyatsky) [inline Streetcode/DbUpdate/Properties/launchSettings.json:11]: is it okay to leave that here?

**Review by Tysyatsky — APPROVED** [review]
It's okay

File: Streetcode/DbUpdate/Properties/launchSettings.json (hunk 4 above)
Comment (ValDekh) [reply Streetcode/DbUpdate/Properties/launchSettings.json]: Old solutions. I've just deleted it.

**Review by wehfis — APPROVED** [review]

**Review by KateYatsiuk — APPROVED** [review]

**Pull request merged**